sdk/formrecognizer/azure-ai-formrecognizer/MIGRATION_GUIDE.md

```diff
@@ -0,0 +1,630 @@
+# Guide for migrating azure-ai-formrecognizer to version 3.2.x from versions 3.1.x and below
+
+This guide is intended to assist in the migration to `azure-ai-formrecognizer (3.2.x)` from versions `3.1.x` and below. It will focus on side-by-side comparisons for similar operations between versions. Please note that version `3.2.0b1` will be used for comparison with `3.1.2`.
+
+Familiarity with `azure-ai-formrecognizer (3.1.x and below)` package is assumed. For those new to the Azure Form Recognizer client library for Python please refer to the [README](readme) rather than this guide.
+
+## Table of Contents
+- [Migration benefits](#migration-benefits)
+- [Important changes](#important-changes)
+    - [Client usage](#client-usage)
+    - [Analyzing document](#analyzing-documents)
+    - [Analyzing a custom model](#analyzing-a-custom-model)
+    - [Training a custom model](#trainig-a-custom-model)
+- [Additional samples](#additional-samples)
+
+## Migration benefits
+
+A natural question to ask when considering whether or not to adopt a new version of the library is what the benefits of doing so would be. As Azure Form Recognizer has matured and been embraced by a more diverse group of developers, we have been focused on learning the patterns and practices to best support developer productivity and add value to our customers.
+
+There are many benefits to using the new design of the `azure-ai-formrecognizer (3.2.x)` library. This new version of the library introduces two new clients `DocumentAnalysisClient` and the `DocumentModelAdministrationClient` with unified methods for analyzing documents and provides support for the new features added by the service in API version `2021-09-30-preview`.
+
+New features provided by the `DocumentAnalysisClient` include having one consolidated function for analyzing general document layout, having a general prebuilt document model type, along with the same prebuilt model types that we included previously (receipts, invoices, business cards, identity documents), and custom models. Moreover, the models used in the newer version of the library, such as `AnalyzeResult`, remove many hierarchical dependencies between document elements and move them to a more top level and easily accesible position. The service has further improved how to define where elements are found on documents by moving towards `BoundingRegion` definitions where an element is returned with the bounding box per page where it is found. Document element fields are returned with more information, such as content and spans. 
```

Comment:
nit: some wording, feel free to ignore or take whatever feedback sounds right to you

```suggestion
New features provided by the `DocumentAnalysisClient` include one consolidated function for analyzing document layout, a general prebuilt document model, along with the same prebuilt models that were included previously (receipts, invoices, business cards, identity documents), and custom models. Moreover, the models introduced in the latest version of the library, such as `AnalyzeResult`, remove hierarchical dependencies between document elements and move them to a more top level and easily accessible position. The service has further improved how to define where elements are located on documents by moving towards `BoundingRegion` definitions allowing for cross-page elements. Document element fields are returned with more information, such as content and spans. 
```